web_app/migrations/versions/4af6caf34144_add_protocol_id_field.py

```diff
@@ -0,0 +1,53 @@
+"""add protocol_id field
```

Comment:
Please, if you can have everything in one migration file, do it. In this case, it's possible. Merge two migration files into one